pkg/prompb/types.pb.go

```diff
@@ -2228,7 +2238,10 @@ func (m *Label) Unmarshal(dAtA []byte) error {
 			if postIndex > l {
 				return io.ErrUnexpectedEOF
 			}
-			m.Name = string(dAtA[iNdEx:postIndex])
+			tmpNameBytes := dAtA[iNdEx:postIndex]
+			m.NameBytes = make([]byte, len(tmpNameBytes))
```

Comment:
Question. We need to keep the bytes so that they don't get GC'd and break `m.Name` or something like that?

Reply:
Right. Since these bytes are coming from HTTP request which we get rid off after unmarshalling.